src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ManagePersonnelPage/AddPersonnelSideSheet/index.tsx

```diff
@@ -22,56 +23,58 @@ const AddPersonnelSideSheet: React.FC<AddPersonnelToSideSheetProps> = ({
     selectedPersonnel,
 }) => {
     const currentContext = useCurrentContext();
-    const { contract, dispatchContractAction } = useContractContext();
+    const currentContract = useContractContext();
     const notification = useNotificationCenter();
     const { formState, setFormState, isFormValid, isFormDirty } = useAddPersonnelForm(
         selectedPersonnel
     );
     const { apiClient } = useAppContext();
-
+    const { basePositions, basePositionsError, isFetchingBasePositions } = useBasePositions()
     const [saveInProgress, setSaveInProgress] = React.useState<boolean>(false);
 
-    const savePersonnelChangesAsync = React.useCallback(async () => {
-        const contractId = contract?.id;
+    const savePersonnelChangesAsync = async () => {
```

Comment:
Merge conflict issue? If, not why remove the callback and try catch implementation?

Reply:
yeah did wrong where when merging a conflict here. 
Added this back in now.